packages/transactions/src/signatures.ts

```diff
@@ -18,18 +18,19 @@ async function getCompiledMessageSignature(message: CompiledMessage, secretKey:
 }
 
 export async function signTransaction<TTransaction extends Parameters<typeof compileMessage>[0]>(
-    keyPair: CryptoKeyPair,
+    keyPairs: CryptoKeyPair[],
```

Comment:
Do you think this is the way to go, or should we offer two function overloads, one with a single and one with an array, and then check which we got?

Reply:
ie. So that you can do both of these.

```ts
signTransaction(signerKeyPair, tx);
signTransaction([signerKeyPairA, signerKeyPairB], tx);
```

Reply:
I could go either way honestly, but I went with the simpler approach.
It's not too much to ask people to bracket a single keypair, but if you think we should do the overloads I am fine with that as well, for some extra UX sugar

Reply:
The way you have it is probably right!